packages/react-error-overlay/build.js

```diff
@@ -5,7 +5,8 @@
  * LICENSE file in the root directory of this source tree.
  */
 const webpack = require('webpack');
-const chalk = require('chalk');
+const Chalk = require('chalk');
+const chalk = new Chalk.constructor({ enabled: true, level: 1 });
```

Comment:
can move this line below the `require` block ?

Reply:
@sendilkumarn yes sure, i'll update it